data/external/scrapers/public_transport.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from external.scraping_utils import cached_json
+import csv
+# CSV indexes
+STATIONID = "stop_id"
+NAME = "stop_name"
+TYPE= "location_type"
+WGS84X = "stop_lat"  # lat
+WSG84Y = "stop_lon"  # lon
+PARENT="parent_station"
+
+def avg(x1, x2):
+    return (x1 + x2) / 2
+
+@cached_json("public_transport.json")
+def scrape_stations():
+    with Path("scrapers/stops.txt").open("r") as file: #the file is from the zip file "Soll-Fahrplandaten" from https://www.mvv-muenchen.de/fahrplanauskunft/fuer-entwickler/opendata/index.html
```

Comment:
The file gets updated only once per year. Since a lot can change on the page in that time, I chose to not automate that. If you still think it is worth to do it automatically though, I can, of course, do that.

Reply:
I think it is worth it.
Rationale: Long-term maintainability